packages/devtools/src/devtoolsdriver.js

```diff
@@ -103,9 +103,11 @@ export default class DevToolsDriver {
             }
 
             this.emit('result', { command, params, retries, result })
-            log.info('RESULT', command.toLowerCase().includes('screenshot')
-                && typeof result === 'string' && result.length > 64
-                ? `${result.substr(0, 61)}...` : result)
+            if (typeof result !== 'undefined') {
```

Comment:
I would like the readability improved here as well by splitting the condition or use variables/functions for naming perhaps

Reply:
created an `isScreenshot` variable to make it more readable.